nautobot-app/hooks/post_gen_project.py

```diff
@@ -15,7 +15,7 @@
 * poetry install
 * poetry shell
 * invoke makemigrations
-* ruff . # this will ensure all python files are formatted correctly, may require `sudo chown -R <my local username> ./` as migrations may be owned by root
+* invoke ruff --fix # this will ensure all python files are formatted correctly, may require `sudo chown -R <my local username> ./` as migrations may be owned by root
```

Comment:
```suggestion
* invoke ruff --fix # this will ensure all python files are formatted correctly, may require `sudo chown -R $USER ./` as migrations may be owned by root
```